flightcheck/appHook.js

```diff
@@ -68,27 +72,19 @@ class AppHook {
     }
   }
 
-  // TODO: clean up whitespace removal code
   issue () {
-    let template = issue.render(`${this.path}/${this.mark}`, this)
+    const template = render(path.join(this.path, this.mark), this)
 
-    template = template.split('\n')
-    let title = template.find(string => string !== '')
-
-    template.splice(0, template.indexOf(title) + 1)
-
-    template = template.join('\n')
+    template.body += `\n<!-- Houston version ${config.houston.version} in ${config.env} environment -->`
```

Comment:
that is an assigned variable from the `package.json` file. So what ever version is in that will be the version outputted. I could try to check for a `.git` folder and try to read the latest commit sha and append that to the version number so it's like `1.0.0-as93jfs` if you think that would be helpful.